omero_marshal/legacy/AffineTransformI.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from math import sin, cos, radians
+from omero.rtypes import unwrap
+
+
+class AffineTransformI(object):
+    """
+    Class handling OMERO 5.1 and OMERO 5.2 transform string representations.
+    Its fields and methods are directly mirroring the
+    omero.model.AffineTransform class introduced in OMERO 5.3 to reduce the
+    complexity of the transform encoders/decoders
+    """
+
+    # Set the id to -1 to differentiate the marshalled transform from objects
```

Comment:
Good catch. Definitely an misuse of variables coming from a straight copy and paste of `omero_model_AffineTransformI.py`. Should be fixed by https://github.com/openmicroscopy/omero-marshal/pull/20/commits/cfdb7f253e7aa34224ec8a2a9d994c3ec4806b33